content/practices/team-shared-history.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2018-06-28T00:00:00-05:00
+title: Team Shared History
+draft: false
+stages: ['delivery']
+---
+
+## What is it?
+
+A method of helping a team start to become cohesive, develop a shared sense of history, and a shared vision of the future. This simple exercise starts with a facilitator organizing the participants in order of their hire date with the company. Then a timeline is drawn and each team member marks their hire date on the timeline. Next, team members note on the timeline interesting things which may have occurred near their time of hire. Finally, the team member who has the most seniority will "tell the story" of the team's history and perhaps embellish with additional context around what has been going on around the organization.
+
+Outcomes
+- A better understanding between team members of their own place in the organization
+- Establish empathy and context for future conversations between team members
+- Build a sense of community for newer team members
+
+## Why Use Team Shared History?
+
+- New team members are not sure of their place, responsibilities, rights, and history of their org.
+- Team members with more history may assume that newer members know what is expected
+
+## Related Practices
+
+
+## Who do you need?
+
+- Facilitator
+- Product Owner
+- Project Sponsor, both technical and business
+- Architects
+- Developers
+- Scrum Master
+
+## Suggested Time
+
+1 hour. The session can be done without any participant prep.
+
+## Difficulty
+
+- Facilitator: Easy
+- Participants: Easy
+
+## Facilitation Materials Needed
+
+A surface on which a timeline can be drawn, writing materials for participants.
```

Comment:
so pencils are ok? If you are going to have "writing materials", may want to be a bit more specific. Think about how it would be very different if you use a whiteboard vs a typical wall and tape. Or if you just write on the surface of your working space versus using post-its.